frontend/src/app/components/modals/create-namespace-bucket-modal/create-namespace-bucket-modal.js

```diff
@@ -97,7 +97,8 @@ class CreateNamespaceBucketModalViewModel extends ConnectableViewModel {
         readPolicy: [],
         writePolicy: undefined,
         cacheTTL: 60000,
-        enableCaching: false
+        enableCaching: false,
+        cacheEvictionType: 'LRU'
```

Comment:
I cannot find any edits on the corresponding view file (the HTML file) that is using this form field. If the view does not use it please remove it or add the change to the view file.

if you remove it all other changes in this are meaningless and should also be removed.
If you just need to send LRU in the dispatch for `createNamespaceBucket` below you don't need all these changes and can just hardcode the `"LRU"` value in the `createNamespaceBucket` call

Reply:
I added a new type config.NAMESPACE_CACHING.DEFAULT_EVICTION_TYPE
This is called in used in `src/server/system_services/bucket_server.js` to set the eviction property when a bucket is created